.gitignore

```diff
@@ -28,6 +28,8 @@ coverage/
 
 # Docker
 .docker-build
+.frontend-build
+.webapp-build
```

Comment:
Should we call them `frontend` and `backend`?

Or `react` and `django`?

Reply:
Whatever the naming system, it ought to be consistent. Which is why I went with `frontend` as the name for the new thing, as that matches the directory.

`react` and `django` feel like they're mixing concerns a bit by referring to the implementation, when we ought to care more about its usage or external API.

Reply:
Good point about avoiding the implementation details in naming conventions.

For container names to be consistent with directory names, we'd also need to rename the top-level `pontoon` folder. Even if not 100% accurate, I'd prefer `backend` over `webapp`. I worry though that renaming the `pontoon` folder will require changes to be made in too many places, and that we might want to split webapp/pontoon further in the future.

So how about we just call the `webapp` container `backend` for now, and leave the directory structure unchanged? That would at least fix the current situation, where `webapp` sounds like something `frontend` is a part of.

Reply:
To bikeshed this a bit a more, would you be ok with `frontend` and `server` containers, rather than calling the latter `backend`? As it is serving the Django parts of the frontend, that might be more a bit more appropriate.

Reply:
Ok, let's do that.